volume/service/store.go

```diff
@@ -195,11 +195,11 @@ func (s *VolumeStore) purge(ctx context.Context, name string) error {
 	if exists {
 		driverName := v.DriverName()
 		if _, err := s.drivers.ReleaseDriver(driverName); err != nil {
-			log.G(ctx).WithError(err).WithField("driver", driverName).Error("Error releasing reference to volume driver")
+			log.G(ctx).WithFields(log.Fields{"error": err, "driver": driverName, "volume-name": name}).Error("Error releasing reference to volume driver")
```

Comment:
nit:
```suggestion
			log.G(ctx).WithFields(log.Fields{logrus.ErrorKey: err, "driver": driverName, "volume-name": name}).Error("Error releasing reference to volume driver")
```

Reply:
Ah, yeah, that's on purpose; trying to avoid direct imports of logrus (we changed all imports to use containerd's logs package, which provides aliases for the essential parts)